graphic_templates/bar_chart/js/graphic.js

```diff
@@ -6,17 +6,18 @@ var isMobile = false;
  * Initialize the graphic.
  */
 var onWindowLoaded = function () {
-    if (Modernizr.svg) {
```

Comment:
Would you mind changing the indentation in this file back to spaces?

I know the .editorconfig file had tabs specified (which was silly given the majority of existing codebase uses spaces) so that might be why this change came about. I've submitted a pull request for .editorconfig here: https://github.com/abcnews/dailygraphics/pull/72 and will merge when it gets a positive review.